client/state/reader/feeds/selectors.js

```diff
@@ -17,7 +17,7 @@ export function shouldFeedBeFetched( state, feedId ) {
 function isStale( state, feedId ) {
 	const lastFetched = state.reader.feeds.lastFetched[ feedId ];
 	if ( ! lastFetched ) {
-		return false;
+		return true;
```

Comment:
i thought it was already like this. Must have changed when we went from `updatedBetweenTimestamp` to `isStale` or something